cogs/pokemon.py

```diff
@@ -683,42 +683,43 @@ async def release(self, ctx, args: commands.Greedy[converters.PokemonConverter])
 
         for pokemon in args:
 
-            if pokemon is None:
-                continue
+            if pokemon is not None:
+                # can't release selected/fav
 
-            # can't release selected/fav
+                if pokemon.id in ids:
+                    continue
 
-            if pokemon.id in ids:
-                continue
+                if member.selected_id == pokemon.id:
+                    await ctx.send(
+                        f"{pokemon.idx}: You can't release your selected pokémon!"
+                    )
+                    continue
 
-            if member.selected_id == pokemon.id:
-                await ctx.send(
-                    f"{pokemon.idx}: You can't release your selected pokémon!"
-                )
-                continue
+                if pokemon.favorite:
+                    await ctx.send(f"{pokemon.idx}: You can't release favorited pokémon!")
+                    continue
 
-            if pokemon.favorite:
-                await ctx.send(f"{pokemon.idx}: You can't release favorited pokémon!")
-                continue
+                ids.add(pokemon.id)
+                mons.append(pokemon)
 
-            ids.add(pokemon.id)
-            mons.append(pokemon)
+        if len(args) != len(mons):    
+            await ctx.send(f"Couldn't find/release {len(args)-len(mons)} pokémon in this selection!")
 
         # Confirmation msg
 
         if len(mons) == 0:
             return
 
-        if len(args) == 1:
+        if len(mons) == 1:
             await ctx.send(
-                f"Are you sure you want to release your level {pokemon.level} {pokemon.species}. No. {pokemon.idx} for 2 pc? This action is irreversible! [y/N]"
+                f'Are you sure you want to **release** your{" ✨" if mons[0].shiny else ""} Level {mons[0].level} {mons[0].iv_percentage * 100:.2f}% {mons[0].species}. No. {mons[0].idx} for 2 pc? [y/N]'
```

Comment:
We can use string formatting to replace this stuff, would probably be better
look around in the auction / market code for examples, I think